cluster/manifests/stackset-controller/01-stack-crd.yaml

```diff
@@ -349,7 +349,7 @@ spec:
                 - maxReplicas
                 - metrics
                 type: object
-{{- if eq .Cluster.ConfigItems.stackset_configmap_support_enabled "true" }}
+{{- if and (eq .Cluster.ConfigItems.stackset_configmap_support_enabled "true") (eq .Cluster.ConfigItems.stackset_secret_support_enabled "true") }}
```

Comment:
This should be or right? one of them can be enabled and it will work. We have configMap enabled in some clusters I assume which would be disabled by default with this change.

Reply:
yes, thanks!